lib/webapp.js

```diff
@@ -75,6 +75,21 @@ function DataserviceContext(serviceDefinition, serviceConfiguration,
   this.serviceDefinition = serviceDefinition;
   this.serviceConfiguration = serviceConfiguration;
   this.plugin = pluginContext;
+
+  /* By default, we init storage object to a default one as to not slow down app init by
+  waiting for the real one (or if we are not in cluster mode) */
+  this.storage = zluxUtil.DataserviceStorage();
+
+  if (process.clusterManager) { // Cluster mode
+    createDataserviceStorage().then((data) => {
+      /* Then, once the cluster is done creating its own storage, we merge the local one
```

Comment:
The idea of needing to merge local & cluster storage can be entirely avoided if there is code guarantees that the cluster finishes its creation prior to this section running. It probably does, but a solution to ensure it would simplify.

Reply:
The cluster does not finish its creation prior to this section running and it would never have the storage that could have been added while the cluster didn't finish loading yet. That was the point of merging them together because at this point, there is
1. the master storage in cluster has loaded
2. the master storage in cluster probably does not have any of the storage updates of the local storage, if the local storage was messed with prior to the cluster finishing loading.

This was all set up to NOT wait for the cluster, since that would slow down app init, am I misunderstanding? Unless we do want to make apps load slower by having wait for the cluster @1000TurquoisePogs